zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/X509AuthenticationUtil.java

```diff
@@ -174,39 +173,32 @@ private static String matchAndExtractSAN(X509Certificate clientCert)
     String extractRegex = X509AuthenticationConfig.getInstance().getClientCertIdSanExtractRegex();
     int extractMatcherGroupIndex =
         X509AuthenticationConfig.getInstance().getClientCertIdSanExtractMatcherGroupIndex();
-    LOG.info("X509AuthenticationUtil::matchAndExtractSAN(): Using SAN in the client cert "
-            + "for client ID! matchType: {}, matchRegex: {}, extractRegex: {}, "
+    LOG.debug("Using SAN from client cert to extract client ID. matchType: {}, matchRegex: {}, extractRegex: {}, "
             + "extractMatcherGroupIndex: {}", matchType, matchRegex, extractRegex,
         extractMatcherGroupIndex);
     if (matchRegex == null || extractRegex == null || matchType < 0 || matchType > 8) {
       // SAN extension must be in the range of [0, 8].
-      // See GeneralName object defined in RFC 5280 (The ASN.1 definition of the
-      // SubjectAltName extension)
-      String errStr = "X509AuthenticationUtil::matchAndExtractSAN(): ClientCert ID type "
-          + "SAN was provided but matchType or matchRegex given is invalid! matchType: " + matchType
-          + " matchRegex: " + matchRegex;
+      // See GeneralName object defined in RFC 5280 (The ASN.1 definition of the SubjectAltName extension)
+      String errStr = "Client cert ID type 'SAN' was provided but matchType or matchRegex given is invalid! "
+          + "matchType: " + matchType + " matchRegex: " + matchRegex;
       LOG.error(errStr);
       throw new IllegalArgumentException(errStr);
     }
     // filter by match type and match regex
-    LOG.info("X509AuthenticationUtil::matchAndExtractSAN(): number of SAN entries found in"
-        + " clientCert: " + clientCert.getSubjectAlternativeNames().size());
+    LOG.debug("Number of SAN entries found in client cert: " + clientCert.getSubjectAlternativeNames().size());
     Pattern matchPattern = Pattern.compile(matchRegex);
     Collection<List<?>> matched = clientCert.getSubjectAlternativeNames().stream().filter(
         list -> list.get(0).equals(matchType) && matchPattern.matcher((CharSequence) list.get(1))
             .find()).collect(Collectors.toList());
 
-    LOG.info(
-        "X509AuthenticationUtil::matchAndExtractSAN(): number of SAN entries matched: " + matched
-            .size() + ". Printing all matches...");
+    LOG.debug("Number of SAN entries matched: " + matched.size() + ". Printing all matches...");
```

Comment:
neat : space before ":". Also, "..." can be removed at the end of string

Reply:
See https://github.com/linkedin/zookeeper/pull/83#issuecomment-1171717208